dashboards/updateserver-dashboard.json

```diff
@@ -24,7 +24,7 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 55,
```

Comment:
Sometimes with the version and id changing the dashboard does not load after redeploying on an environment where the dashboard already existed.